packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -287,7 +288,11 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   _DragState _state = _DragState.ready;
   late OffsetPair _initialPosition;
   late OffsetPair _pendingDragOffset;
+
+  /// The position of the last pointer event received.
+  OffsetPair get finalPosition => _finalPosition;
```

Comment:
Would lastPosition be a better name here?

Reply:
Yeah, probably. Updated.